### PR TITLE
Added layer 2 examples

### PR DESCRIPTION
**Pull request by federicorinaldi**
- Added layer 2 examples
- Updated README file with the new commands and removed unused commands
- Updated package* with the new version of the SDK

### CODE REVIEW

**Review by DZariusz — CHANGES_REQUESTED** [review]
please:
- rebase with develop
- add pipeline for CI (you can just copy the file https://github.com/umbrella-network/phoenix/blob/develop/.github/workflows/pipeline.yml)

File: .github/workflows/pipeline.yml [hunk 1]
```diff
@@ -43,3 +43,23 @@ jobs:
...
+        API_KEY: ${{ secrets.APY_KEY }}
+      run: |
+        npx hardhat run scripts/api-client.ts
+    - name: Run proof verification example
+      env:
+        RPC_URL: ${{ secrets.RPC_URL }}
+        UMB_CHAIN_ADDRESS: ${{ secrets.UMB_CHAIN_ADDRESS }}
+        API_KEY: ${{ secrets.APY_KEY }}
+      run: |
+        npx hardhat run scripts/proof-verification.ts
```
Comment (DZariusz) [inline .github/workflows/pipeline.yml:65]: lets add also `hardhat test`

**Review by DZariusz — APPROVED** [review]
once pass CI lets merge

File: .github/workflows/pipeline.yml (hunk 1 above)
Comment (federicorinaldi) [reply .github/workflows/pipeline.yml:68]: Perfect, added. Once CI passes I will merge and close the PR :)

**Pull request merged**